src/modules/person/domain/person.ts

```diff
@@ -77,6 +78,14 @@ export class Person<WasPersisted extends boolean> {
         return this.passwordInternalState.isTemporary;
     }
 
+    private static validateName(name: string, fieldName: string): Option<DomainError> {
+        const NO_LEADING_TRAILING_WHITESPACE: RegExp = /^(?! ).*(?<! )$/;
+        if (!NO_LEADING_TRAILING_WHITESPACE.test(name) || name.trim().length === 0) {
+            return new NameValidationError(fieldName);
+        }
+        return null;
+    }
+
     public static construct<WasPersisted extends boolean = false>(
```

Comment:
Wir sollten uns vielleicht Gedanken machen dass der Costruct nicht missbraucht wird und dass nur in Repos verwendet wird um die Daten zu mappen, sonnst könnte man die Spezifikation im Aggregate umgehen.